images/algod/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM golang:1.13.12
```

Comment:
you might want to change this to 1.14.7 to align with the existing go-algorand version.

Reply:
Thanks, changed.